packages/sanity/src/structure/panes/document/DocumentPane.tsx

```diff
@@ -115,6 +115,7 @@ function DocumentPaneInner(props: DocumentPaneProviderProps) {
               t={t}
               i18nKey="panes.document-pane.document-not-found.text"
               values={{id: options.id}}
+              components={{Code: ({children}) => <code>{children}</code>}}
```

Comment:
It crashed and complained about Code not being declared instead of showing the error. Then I saw a commit from @rexxars that fixed the same error but from a different location. Hang on let's see if I can find it.

Reply:
https://github.com/sanity-io/sanity/pull/5434

Reply:
oh hmm. I was referencing #5114

> It's a bit tedious to pass in components like <Code> that just maps to <code>. This PR allows using this predefined list of tags without specifying it directly: <abbr>, <address>, <cite>, <code>, <del>, <em>, <ins>, <kbd>, <q>, <samp>, <strong>, <sub>, <sup>.

the suggestion is to change any translation string to remove `<Code>` in favor of just `<code>` instead of adding support for `<Code>` but maybe we don't have control over those translation strings?

nbd though

Reply:
Gotcha, I have to admit I'm not too familiar with how our i18n works yet. All I know is that adding this line allowed the 404 to render again, instead of crashing the Structure Tool and propagating to the `StudioErrorBoundary` that wraps the tool layout :D